hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -399,15 +418,19 @@ Instant getConsensusTimeOfLastHandledTxn() {
 		return consensusTimeOfLastHandledTxn;
 	}
 
-	DeterministicThrottle.UsageSnapshot[] getUsageSnapshots() {
-		return usageSnapshots;
-	}
-
 	void setUsageSnapshots(DeterministicThrottle.UsageSnapshot[] usageSnapshots) {
 		this.usageSnapshots = usageSnapshots;
 	}
 
 	DeterministicThrottle.UsageSnapshot[] usageSnapshots() {
```

Comment:
nit : getUsageSnapshots() would be more appropriate to keep and delete this method.